docs/user/custom-resources/apirule/04-10-apirule-custom-resource.md

```diff
@@ -70,6 +70,9 @@ The following status codes describe VirtualServices and Oathkeeper Access Rules:
 | **SKIPPED** | Skipped creating a resource. |
 | **ERROR**   | Resource not created.        |
 
+## Access Strategies
```

Comment:
🤔 Maybe this can be removed since we already have a separate page for access strategies?